consensus/validator.go

```diff
@@ -196,6 +208,62 @@ func (c *Consensus) validateBlockBody(blk *block.Block) error {
 		}
 	}
 
+	bss := blk.BackerSignatures()
+	if header.BackerSignaturesRoot() != bss.RootHash() {
+		return consensusError(fmt.Sprintf("block backers root mismatch: want %v, have %v", header.BackerSignaturesRoot(), bss.RootHash()))
+	}
+
+	totalBackers := uint64(len(bss)) + parent.TotalBackersCount()
+	if header.TotalBackersCount() != totalBackers {
+		return consensusError(fmt.Sprintf("block total backers count invalid: want %v, have %v", header.TotalBackersCount(), totalBackers))
+	}
+
```

Comment:
discussion: is there a window that one can build a vip193 block to split the network? if so, how to prevent it